.rubocop.yml

```diff
@@ -0,0 +1,153 @@
+#                           when 1 == 1
+#                             do_something
+#                           else
+#                             do_else
+#                           end
+#
+Lint/EndAlignment:
+  AlignWith: variable
+
+# [codesmell]
+Metrics/AbcSize:
+  Enabled: false
+  Exclude:
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+
+# [codesmell]
+Metrics/ClassLength:
+  Enabled: false
+  Exclude:
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+
+# [codesmell]
+Metrics/MethodLength:
+  Enabled: false
+  Exclude:
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+  Max: 15
+
+# [codesmell]
+Metrics/LineLength:
+  Enabled: false
+  Exclude:
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+  Max: 100
+
+# [codesmell]
+# I don't really get the point of this cop.
+Performance/RedundantMerge:
+  Enabled: false
+
+# Do not use "and" or "or" in conditionals, but for readability we can use it
+# to chain executions. Just beware of operator order.
+Style/AndOr:
+  EnforcedStyle: conditionals
+
+# braces_for_chaining seems a good fit of what we've been doing so far.
+Style/BlockDelimiters:
+  EnforcedStyle: braces_for_chaining
+  IgnoredMethods:
+    - expect
+
+# Double empty lines are useful to separate conceptually different methods
+# in the same class or module.
+Style/EmptyLines:
+  Enabled: false
+
+Style/EmptyLinesAroundBlockBody:
+  Exclude:
+    # RSpec is all made of blocks. Disable this config in RSpec
+    # to be consistent with EmptyLinesAroundClassBody and EmptyLinesAroundModuleBody
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+
+# In most cases, a space is nice. Sometimes, it's not.
+# Just be consistent with the rest of the surrounding code.
+Style/EmptyLinesAroundClassBody:
+  Enabled: false
+
+# In most cases, a space is nice. Sometimes, it's not.
+# Just be consistent with the rest of the surrounding code.
+Style/EmptyLinesAroundModuleBody:
+  Enabled: false
+
+# I personally don't care about the format style.
+# In most cases I like to use %, but not at the point I want to enforce it
+# as a convention in the entire code.
+Style/FormatString:
+  Enabled: false
+
+# Prefer the latest Hash syntax
+Style/HashSyntax:
+  Exclude:
+    # But Rakefiles generally have some definition like
+    #   :default => :test
+    # that looks nicer with the old rocket syntax.
+    - 'Rakefile'
+
+# Array indentation should be considered like MultilineMethodCallIndentation indentation
+# and use 4 spaces instead of 2.
+Style/IndentArray:
+  IndentationWidth: 4
+
+# Hash indentation should be considered like MultilineMethodCallIndentation indentation
+# and use 4 spaces instead of 2.
+Style/IndentHash:
+  IndentationWidth: 4
+
+# [codesmell]
+# It's not always that bad.
+Style/IfInsideElse:
+  Enabled: false
+
+# Multi-line differs from standard indentation, they are indented twice.
+Style/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+  IndentationWidth: 4
```

Comment:
Yes, correct. In general, line continuation (or multiline) is indented twice. You can change the cop to 2 to see the various affected code fragments.

It's actually more evident in the main DNSimple app.

Another example is

```ruby
this_is_something("to be done with a successful result") or
    raise("something really bad happened")
```

You will see a lot of examples in the DNSimple app. I'm not surprised that we have never documented it explicitly, that's one of the various conventions that I personally adopted while developing the codebase (I can't even recall where I got that) but I've never enforced.

This is also why I'd like to start coding the various habits in a tool. To make sure we agree on something, and we have a tool that helps us to document it (especially for new devs).

Given I've used this convention for a while, how do you feel about it? Are you ok to adopt it?